index.bs

```diff
@@ -837,11 +863,23 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 						
 						1. Set |representation|'s [=representation/MIME type=] to |mimeType|.
 
+						1. Let |isUnsanitized| be |false|.
+
+						1. If |formats| is not empty, then:
+
+							1. For each |format| in |formats|["{{ClipboardUnsanitizedFormats/unsanitized}}"]:
+
+								1. If |format| is equal to [=representation/MIME type=], set |isUnsanitized| to true.
+
 						1. Set |representation|'s [=representation/data=] to |systemClipboardRepresentation|'s [=system clipboard representation/data=].
 
 							Issue: It should be possible to read the data asynchronously from the system clipboard after the author calls getType, however, this set of steps implies that data will be provided at the time of read.
 						
-						1. The user agent, MAY sanitize |representation|'s [=representation/data=], unless |representation|'s [=representation/MIME type=]'s essence is "image/png", which should remain unsanitized to preserve meta data.
+						1. The user agent, MUST NOT sanitize |representation|'s [=representation/data=], if it satisfies the below conditions:
+						
+							1. |representation|'s [=representation/MIME type=]'s [=MIME type/essence=] is "image/png", which should remain unsanitized to preserve meta data.
```

Comment:
Should we include this condition in the definition of "unsanitized data types" above? Something like: Unsanitized data types are "image/png" and those that are passed by the user as part of the unsanitized option.

Reply:
Yeah, this is something that I wasn't sure about. I don't know the spec language for it, but I wanted to write something that allows MIME types in `unsanitized data types` list to be part of `{{ClipboardUnsanitizedFormats/unsanitized}}`, but some MIME types(like `image/png`) should be treated like `unsanitized data types` and doesn't have to be in `{{ClipboardUnsanitizedFormats/unsanitized}}`.

Currently Webkit and Chromium allow unsanitized `image/png` read/write by-default so web authors don't have to specify it in the new `{{ClipboardUnsanitizedFormats/unsanitized}}` option during `read()`.

Reply:
Clarified few things.